docs/6/6.8-eks.md

```diff
@@ -97,13 +97,25 @@ In the previous section we launch the SockShop demo on an ECS cluster to demonst
 
 ?> Make sure to use an EC2 instance type with enough CPU and memory resources for the each nodes overhead and the pods workload (t2.micro will not be enough). Look through the different [EC2 instance types](https://aws.amazon.com/ec2/instance-types/)
  
-5. Specify your launch template when creating a Node Group to launch Kubernetes EC2 images
+5. Specify your launch template when creating a Node Group to launch Kubernetes EC2 images. Set max nodes to 4, desired nodes to 3, and minimum nodes to 2.
 6. Launch SockShop application on your cluster using the `complete-demo.yaml` in the [microservice-demo](https://github.com/liatrio/microservices-demo) repo
 
 ?> Depending on the version of Kubernetes you select when creating your EKS cluster, the `complete-demo.yaml` may need to be updated.
 
+## Exercise - Adding an autoscaler to your cluster
+
+Now you will add an autoscaler to your cluster that will automatically scale your Node Group up and down depending on the capacity of the nodes in the group.
+
+1. Read about the Kubernetes autoscaler [here](https://docs.aws.amazon.com/eks/latest/userguide/cluster-autoscaler.html)
+2. Using `kubectl` or `k9s` check that there are 3 nodes running in your Node Group
+3. Follow the AWS documentation to [add a cluster autoscaler](https://docs.aws.amazon.com/eks/latest/userguide/cluster-autoscaler.html) to the cluster you created in the previous exercise.
+  ?> Pay particular attention to the policy you are adding to your Node Groups IAM Role
+4. Verify that there is a `cluster-autoscaler` running in your `kube-system` namespace. Check the logs to see what it is doing.
+5. Check that the autoscaler successfully scales your Node Group down to 2 instances.
+
 ## Deliverables
 
 - Discuss the differences between ECS and EKS
 - Explain how services remedy the difficulties of ephemeral pods
 - Describe the benefits/drawbacks of using EKS in enterprise
+- Discuss why the policy added in the autoscaler exercise might be a bad idea ([this is an alternative](https://aws.amazon.com/blogs/opensource/introducing-fine-grained-iam-roles-service-accounts/))
```

Comment:
I'd clarify that you mean an IAM policy, and possibly reword this sentence a bit to avoid having the addendum at the end.